docs/guides/integration-javascript.md

```diff
@@ -0,0 +1,293 @@
+---
+title: Javascript Express Integration
```

Comment:
Javascript -> JavaScript. Same comment for all other usages of "Javascript"

Reply:
fixed